src/common/mmo.h

```diff
@@ -57,10 +57,19 @@ enum FLAGTYPE : uint32
     FLAG_GM_LEAD        = 0x06000000,
     FLAG_GM_PRODUCER    = 0x07000000,
     FLAG_BAZAAR         = 0x80000000,
+};
 
-//  FLAG_AUTOGROUP      = 0x00000000,
-//  FLAG_NEWPLAYER      = 0x00000000,
-//  FLAG_MENTOR         = 0x00000000,
+enum NFLAGTYPE : uint32
```

Comment:
>  Figure out better name for unnamedFlags

maybe "flags"/"nameflags"(same thing) and "unnamedFlags" could be "entityFlags" and "playerFlags" respectively. The former have near identical analogs in the flags used on mobs and npc but the later are player specific at least in usage (I wouldn't be surprised if you could force them into non players, I can already make GM goblins and bazaar icon's leaping lizzies.)